tests/functional/test_install_wheel.py

```diff
@@ -60,6 +60,22 @@ def test_basic_install_from_wheel(script, shared_data, tmpdir):
     assert script_file in result.files_created
 
 
+def test_basic_install_from_unicode_wheel(script, data):
+    """
+    Test installing from a wheel containing non-ASCII file names.
+    """
+    result = script.pip(
+        'install', 'test_unicode==1.0', '--no-index',
+        '--find-links=' + data.find_links,
```

Comment:
You might be able to create the wheel with `create_basic_wheel_for_package` helper function. This would remove the need to copy the whole data directory.
Using `shared_data` fixture is an other solution but generating the wheel in the test case seems cleaner & clearer.

Reply:
Thanks! didn’t know that was a thing.